hardware/raspberrypi/bootmodes/nvme.md

```diff
@@ -135,22 +134,22 @@ enable_uart=1
 uart_2ndstage=1
 ```
 
-Check you have the right version of the start4.elf VideoCore firmware by checking the version of the raspberrypi-bootloader package, it should show a date on or after 3 Mar 2021 in the version string. Run `apt show raspberrypi-bootloader`
+Check you have the right version of the `start4.elf` VideoCore firmware by checking the version of the raspberrypi-bootloader package: it must show a date of 3 Mar 2021 or later in the version string. Run `apt show raspberrypi-bootloader`:
```

Comment:
Hmmm, isn't `apt show raspberrypi-bootloader` the wrong thing to do if `rpi-update` is being used? :thinking:

Reply:
Good point. I've rewritten that and the few paragraphs after that which talk about the kernel version as well.